packages/kit/src/head.ts

```diff
@@ -0,0 +1,8 @@
+import type { NuxtAppConfig } from '@nuxt/schema'
+import { useNuxt } from './context'
+import { defu } from 'defu'
+
+export function addAppHead (head: NuxtAppConfig['head']) {
```

Comment:
I think we probably need a different name to communicate that this will set global values (but can be overridden). what about `setDefaultHead`?